final_task/rss_reader/converter.py

```diff
@@ -0,0 +1,152 @@
+
+from dominate.tags import div, h2, img, p, a, meta
+from datetime import datetime
+import dominate
+import html
+import os
+import httplib2
+import socket
+from colorizer import printerr
+from fpdf import FPDF
+
+def is_connected() -> bool:
+    try:
+        socket.create_connection(("www.google.com", 80))
+    except Exception:
+        return False
+    return True
+
+
+def font_decorator(function_to_decorate):
+    """
+    The wrapper takes the function that sets the font
+    and applies this font to the text
+    """
+    def wrapper(text, page, font):
+        function_to_decorate(text, page, font)
+        if font == 'Arial':
+            page.cell(1, 1, text + '\n')
+        else:
+            page.multi_cell(w=100, h=6, txt='\n' + text + '\n\n', align='L')
+    return wrapper
+
+
+@font_decorator
+def font_print(text, page, font):
+    """
+    Sets a font on a page
+    Has a text argument to have a possibility to be wrapped
+    """
+    if font == 'Arial':
+        page.set_font(font, 'B', size=12)
+    else:
+        page.set_font(font)
+
+
+def pdf_convert(news, limit, path, color):
```

Comment:
This function is really large, it is better to split it somehow into the smaller ones.